spec/views/hq/orphans/_form_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+require 'cgi'
+
+RSpec.describe "hq/orphans/_form.html.erb", type: :view do
```

Comment:
Can we have a discussion again about the need for this extensive testing of the form? Rails is handling pre-population of fields with the correct data. Are we double-checking everything just to make sure that we didn't make any typos?